### ISSUE TITLE
programmatically tell if ontology has `treat-xrefs-as-equivalent` set, using `ontobio`?

### ISSUE THREAD

**Issue by saramsey** [RTXteam/RTX#413]
Would like to know if it is possible to determine programmatically, using the ontobio API, whether an ontology that I have loaded has (in its document header) the setting `treat-xrefs-as-equivalent`. Have posed this question to Chris Mungall.

**Comment by saramsey** [comment]
This information would be helpful because then we can use the stronger owl:sameAs predicate instead of the semantically loose `xref`.

**Comment by saramsey** [comment]
Actually Chris Mungall says:

> I would not depend on that tag in any way
> 
> Making sense of xrefs is a fraught business. You can read my ramblings on this matter here:
> https://douroucouli.wordpress.com/2019/05/27/never-mind-the-logix-taming-the-semantic-anarchy-of-mappings-in-ontologie/
> 
> TL;DR for inter-OBO linkages you want to be using explicit logical axioms like equivalentClass (of which there should be few, since OBO is intended to be orthogonal). You also have to be careful with nbo, it's a bit abandoned - do you need equivalencies between GO and NBO classes?
> 
> You may also want to look at the monarch.owl mega-importer where we attempt to include all the right bridge modules and plug as necessary.. deepak we should probably have a standard workflow for this
